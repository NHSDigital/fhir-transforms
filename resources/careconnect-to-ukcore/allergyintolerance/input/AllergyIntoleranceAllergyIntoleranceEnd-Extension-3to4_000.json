{
  "resourceType": "AllergyIntolerance",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-AllergyIntoleranceEnd-1",
      "extension": [
        {
          "url": "endDate",
          "valueDateTime": "2016-11-01T00:00:00+00:00"
        },
        {
          "url": "reasonEnded",
          "valueString": "Desensitised to Peanuts"
        }
      ]
    }
  ]
}
