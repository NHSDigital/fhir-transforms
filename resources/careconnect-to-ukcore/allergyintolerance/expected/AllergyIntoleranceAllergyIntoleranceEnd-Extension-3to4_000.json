{
  "resourceType": "AllergyIntolerance",
  "extension": [
    {
      "extension": [
        {
          "url": "endDate",
          "valueDateTime": "2016-11-01T00:00:00+00:00"
        },
        {
          "url": "reasonEnded",
          "valueString": "Desensitised to Peanuts"
        }
      ],
      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-AllergyIntoleranceEnd"
    }
  ]
}